deepecho/models/basic_gan.py

```diff
@@ -169,8 +169,6 @@ def __init__(self, epochs=1024, latent_size=32, hidden_size=16,
         self._verbose = verbose
 
         LOGGER.info('%s instance created', self)
-        if verbose:
```

Comment:
Maybe we could do something like
```
if verbose:
     LOGGER.setLevel(INFO)
```
so that we still get the `instance created` logs on verbose

Reply:
That would not be recommended. A library like this one should only send messages to the logger, specifying the message level (like the `LOGGER.info(<message>)`) but it should never interact with the logging configuration such as the logging handlers or the logger levels. This is something that will be configured at the application level or by the end user.

Reply:
That makes sense. I'm a bit confused about when we would use print vs when we would use the `LOGGER.info`. Would users have to enable a logging level of `INFO` and set `verbose=True` to see all the prints?